modules/ROOT/pages/index.adoc

```diff
@@ -23,22 +23,25 @@ Understanding these dependencies has powerful implications, especially when you
 
 The following use cases provide examples of how you can use Anypoint Visualizer and its various visualizations.
 
-* Discussing Current Design State: Enterprise architect Kara has developed a plan for how the architecture she’s responsible for will accommodate a new product. She needs to capture its current state so she can plan the transition and get buy-in from stakeholders on her proposal. She can use the Architecture visualization to assess the structure of her network and export it to share with her audience.
+* Discussing Current Design State +
+Enterprise architect Kara has developed a plan for how the architecture she’s responsible for will accommodate a new product. She needs to capture its current state so she can plan the transition and get buy-in from stakeholders on her proposal. She can use the Architecture visualization to assess the structure of her network and export it to share with her audience.
```

Comment:
It is not clear what "it" refers to Is it the Architecture visualization diagram?

Reply:
```suggestion
Enterprise architect Kara has developed a plan for how the architecture she’s responsible for will accommodate a new product. She needs to capture its current state so she can plan the transition and get buy-in from stakeholders on her proposal. She can use the Architecture visualization to assess the structure of her network and export a diagram to share with her audience.
```